README.md

```diff
@@ -98,6 +98,18 @@ First, install it: `npm install --save-dev aliasify`
 }
```

Comment:
think it'd be possible to add `"global":true` to the package.json version above too? or is the global option only exposed when using the transform via Browserify's Node API?

Reply:
I tried to see if that works just in case but the documentation seems to say you can only use the global transform option from the api or the cli:
https://github.com/browserify/browserify#btransformtr-opts
"[...] You can also not configure global transforms in a package.json like you can with ordinary transforms."

Reply:
aw :( I wonder why they don't allow that